src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+    <Project Include="System.Private.ServiceModel.csproj">
```

Comment:
We still have to workout the full OS support graph but for now I would say only add the entries for the OS's you distinctly need to build for. If it is just Windows vs Unix, I would just have the Windows and add Linux as a representative for Unix for now. We will likely add a Unix flavor at some point.